src/main/java/spireMapOverhaul/zones/volatileGrounds/monsters/UnstableSunstone.java

```diff
@@ -0,0 +1,107 @@
+package spireMapOverhaul.zones.volatileGrounds.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateFastAttackAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.status.Burn;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.volatileGrounds.powers.EruptPower;
+import spireMapOverhaul.zones.volatileGrounds.powers.UnstablePower;
+
+public class UnstableSunstone extends CustomMonster {
+    public static final String ID = SpireAnniversary6Mod.makeID("UnstableSunstone");
+    private static final MonsterStrings monsterStrings = CardCrawlGame.languagePack.getMonsterStrings(ID);
+    public static final String NAME = monsterStrings.NAME;
+    public static final String[] MOVES = monsterStrings.MOVES;
+    private static final String IMG = SpireAnniversary6Mod.makeImagePath("monsters/UnstableSunstone/skeleton.png");
+    private static final String ATLAS = SpireAnniversary6Mod.makeImagePath("monsters/UnstableSunstone/skeleton.atlas");
+    private static final String SKELETON = SpireAnniversary6Mod.makeImagePath("monsters/UnstableSunstone/skeleton.json");
+    private static final byte BUFF = 0;
+    private static final byte ATTACK1 = 1;
+    private static final byte ATTACK2 = 2;
+    private static final int ATTACK1_DAMAGE = 10;
+    private static final int ATTACK2_DAMAGE = 5;
```

Comment:
The damage for these attacks should increase on A3. Probably to 12 and 6 respectively.